src/frontend/next/src/pages/layouts/MDXPageBase.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * A base for building all of our MDX pages.  Any other "global"
+ * things that need to happen for a page should get added here,
+ * and then they will trickle down to the other MDX pages.
+ *
+ * Make sure any new MDX style pages use this to wrap all
+ * elements in their render function.
+ */
+
+import { makeStyles } from '@material-ui/core/styles';
+
+type MDXPageBaseProps = {
+  children: any;
+  // title: PropTypes.string.isRequired,
```

Comment:
This comment was an oversight but I needed to ask about what type the `children` pages should be in this scenario anyway. Does anyone know what this could be? I tried to search this up but I'm not even sure what to research for this.

Reply:
`PropTypes.object.isRequired`, maybe?

Reply:
`children: object;` seems to work!

Reply:
Avoid propTypes if you can. We're using TS so we should take advantage of that

Reply:
@chrispinkney : I don't think you should use `object` here. From the look of it. It looks like a React component to me here. 
```
<MDXPageBase title={'About Us'}>
        <About />
</MDXPageBase>
```
Have you try `children: React.ReactNode`?